_data/_wmt_events_schema.json

```diff
@@ -0,0 +1,253 @@
+    },
+    "links":{
+      "type":"array",
+      "items":{
+        "type":"string"
+      },
+      "minItems":1
+    },
+    "importantDates":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "properties":{
+          "name":{
+            "type":"string"
+          },
+          "date":{
+            "type":"string"
+          }
+        }
+      },
+      "minItems":1
+    },
+    "sharedTasks":{
+      "type":"array",
+      "items":{
+        "type":"string"
+      },
+      "minItems":1
+    },
+    "generalMachineTranslation":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "properties":{
+          "title":{
+            "type":"string"
+          },
+          "description":{
+            "type":"string"
+          }
+        }
+      },
+      "minItems":1
+    },
+    "languagePairs":{
+      "type":"array",
+      "items":{
+        "type":"string"
+      },
+      "minItems":1
+    },
+    "scientificPapers":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "properties":{
+          "topics":{
+            "type":"array",
+            "items":{
+              "type":"string"
+            }
+          },
+          "researchPapers":{
+            "type":"array",
+            "items":{
+              "type":"string"
+            }
+          },
+          "systemPapers":{
+            "type":"array",
+            "items":{
+              "type":"string"
+            }
+          }
+        }
+      },
+      "minItems":1
+    },
+    "paperSubmission":{
+      "type":"array",
+      "items":{
+        "type":"string"
+      },
+      "minItems":1
+    },
+    "posterFormat":{
+      "type":"array",
+      "items":{
+        "type":"string"
+      },
+      "minItems":1
+    },
+    "multidaySchedule":{
+      "type":"array",
+      "items":{
+        "type":"array",
+        "items":{
+          "type":"object",
+          "properties":{
+            "time":{
+              "type":"string"
+            },
+            "title":{
+              "type":"string"
+            }
+          }
+        }
+      },
+      "minItems":1
+    },
+    "oneDaySchedule":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "properties":{
+          "time":{
+            "type":"string"
+          },
+          "title":{
+            "type":"string"
+          }
+        }
+      },
+      "minItems":1
+    },
+    "results":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "properties":{
+          "fullResults":{
+            "type":"object",
+            "properties":{
+              "title":{
+                "type":"string"
+              },
+              "url":{
+                "type":"string"
+              }
+            }
+          },
+          "scoringTable":{
+            "type":"array",
+            "items":{
+              "type":"object",
+              "properties":{
+                "language":{
```

Comment:
do we need `language` if we have `language_pair`?

Reply:
I think so because the table is for one language but with multiple language pairs. 
![Screenshot from 2024-02-08 15-26-37](https://github.com/machinetranslate/machinetranslate.org/assets/94985882/f0e80267-a049-4c83-b9d9-d2f19e678be4)

In the above screenshot, the language is English, and the language pairs are inside the Language Pairs columns. 

What do you think?